calico_kubernetes/calico_kubernetes.py

```diff
@@ -97,6 +147,223 @@ def _configure_interface(self):
         print('Finished configuring network interface')
         return ep
 
+    def _configure_profile(self, endpoint):
```

Comment:
Still not 100% sure why this has moved - could you stick _configure_profile back up at line 85 where it belongs? :)